tests/core/test_bungiesearch.py

```diff
@@ -279,7 +279,10 @@ def test_time_indexing(self):
         except Exception as e:
```

Comment:
Yes, I think this would be a better way of writing this, now that you mention it.